R/RACDPAL.R

```diff
@@ -0,0 +1,67 @@
+#' @title Participation and Activity Limitation 
```

Comment:
"Limitations"? instead of "Limitation"

Reply:
Limitation was used across the CCHS cycles.